main.py

```diff
@@ -1,12 +1,26 @@
 from fetch_last_webpage import fetch_last_webpage, mydb, save_new_version
 from fetch_current_webpage import fetch_current_webpage
+from compare_webpages import compare_webpages
+from generate_message import generate_message
 import discord
 import os
 import asyncio
 
 URL_TO_FETCH = 'https://www.bcm.edu/education/school-of-medicine/m-d-program/current-students/student-affairs/class-of-2025'
 
+def run_bot():
+    curr = fetch_current_webpage(URL_TO_FETCH)
+    prev = []#fetch_last_webpage()
```

Comment:
I don't see `fetch_last_webpage()` being called anywhere. Was that intentional?

Reply:
good catch. i forgot to uncomment that after testing the diff.